config/methods.php

```diff
@@ -270,7 +270,11 @@
 		 * Turns the field value into an absolute Url
 		 */
 		'toUrl' => function (Field $field): string|null {
-			return $field->isNotEmpty() ? Url::to($field->value) : null;
+			try {
+				return $field->isNotEmpty() ? Url::to($field->value) : null;
+			} catch (Throwable) {
```

Comment:
To avoid swallowing other kinds of errors:

```suggestion
			} catch (NotFoundException) {
```

Might need `use` at the top of the file.